.pre-commit-config.yaml

```diff
@@ -0,0 +1,12 @@
+repos:
+-   repo: https://github.com/psf/black
+    rev: 23.1.0
+    hooks:
+    -   id: black
```

Comment:
does this autoformat on precommit? or just reveal errors

Reply:
Auto-format. Would you prefer it behave differently?

Reply:
sounds good to me!